### PR TITLE
docs(UserGuide): update search command section

### PR DESCRIPTION
**Pull request by kyuichyi**
# Checklist for feature implementation and modification
Closes #<Issue_Number>
- [ ] Added relevant milestone (under milestone tab on the right)
- Implementation
  - [ ] Sources (if any) credited under both relevant section(s) and Developer Guide (Acknowledgement)
  - [ ] JUnit Test(s)
- Documentation
  - [ ] JavaDoc for classes and methods
  - Developer Guide
    - [ ] Design (Can add as a todo statement for later on)
    - [ ] Implementation (Can add as a todo statement for later on)
    - [ ] Relevant User Story present
    - [ ] Non-functional requirement(s) that the feature addresses (if any)
    - [ ] Glossary of the (technical) term(s) used (if any)
    - [ ] Specialised instructions for manual testing (if any)
  - User Guide
    - [ ] Description and usage of feature (if any)
  - Contributions (under `/docs/team/*.md`)
    - [ ] Code/Feature/Enhancement contribution
    - [ ] Developer Guide contribution
    - [ ] User Guide contribution
    - [ ] Misc and team-based admin tasks (if any)

# Checklist for non-implementation related PR
Closes #<Issue_Number>
- [ ] Added relevant milestone (under milestone tab on the right)
- Documentation (Contributions)
  - [ ] Developer Guide contributions (if any)
  - [x] User Guide contributions (if any)
  - [ ] Review contributions (if any)
  - [ ] Miscellaneous and team-based admin tasks (if any)

# Additional Information

Nil

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -250,6 +250,33 @@ SUCCESS: Medicine has been deleted
...
 
+To search for a medication, you can type 'search', followed by the field you want to search for and the keyword 
+`keyword`.
+You have to include at least one of the following flags: `-n`, `-i`, `-s`, `-a` to specify the field you want to 
+search for.
+
+```
+Usage:
+    search ([-n name] [-i illness] [-s sideEffects] [-a allFields] keyword) [-h]
+Options:
```
Comment (nickczh) [inline docs/UserGuide.md:261]: Would leaving a line before Options increase readability?

**Review by nickczh — CHANGES_REQUESTED** [review]
Just a small nit! Otherwise LGTM.

File: docs/UserGuide.md (hunk 1 above)
Comment (kyuichyi) [reply docs/UserGuide.md]: resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/142/commits/415b6935cb0b940f93876e6598e7cf97ec419f0c

**Review by nickczh — APPROVED** [review]
LGTM!

**Pull request merged**